chef/cookbooks/crowbar/files/default/crowbar.service

```diff
@@ -1,15 +1,29 @@
 [Unit]
-Description=Crowbar Daemon
+Description=Crowbar
 After=network.target syslog.target remote-fs.target chef-server.service
 
 [Service]
 Type=simple
 User=crowbar
 Group=crowbar
-PIDFile=/var/run/crowbar/crowbar.pid
+KillMode=control-group
+TimeoutSec=300
+TimeoutStopSec=10
+
+WorkingDirectory=/opt/dell/crowbar_framework
+
+Environment="CROWBAR_ENV=production"
+Environment="CROWBAR_THREADS=1"
+Environment="CROWBAR_WORKERS=5"
```

Comment:
Just checking, because I had understood that we wanted 2 threads and 4 workers: @tboerger do we want 1 thread and 5 workers by default?